apps/sasquatch/src/main/java/com/microsoft/azure/mobile/sasquatch/activities/MainActivity.java

```diff
@@ -55,16 +56,15 @@ protected void onCreate(Bundle savedInstanceState) {
         MobileCenter.start(getApplication(), getAppSecret(), Analytics.class, Crashes.class);
 
         Log.i(LOG_TAG, "Crashes.hasCrashedInLastSession=" + Crashes.hasCrashedInLastSession());
-        new AsyncTask<Void, Void, Void>() {
+        Crashes.getLastSessionCrashReport(new ResultCallback<ErrorReport>() {
```

Comment:
You should wait for new binaries for this change. We don't have this method available in jcenter.